params/config.go

```diff
@@ -531,6 +531,14 @@ type ScrollConfig struct {
 
 	// L1 config
 	L1Config *L1Config `json:"l1Config,omitempty"`
+
+	// System contract config
+	SystemTxConfig *SystemTxConfig `json:"systemTxConfig,omitempty"`
+}
+
+type SystemTxConfig struct {
+	Signers   []common.Address `json:"signers"`
+	Contracts []common.Address `json:"contracts"`
```

Comment:
Same for `Contracts`, if it's in the config it will be hard to extend this list in the future.

Reply:
agreed, we should probably put it state.